### PR TITLE
Config fixes

### PR DESCRIPTION
**Pull request by make-sundae**
Fixed backend error on candidate position when rearranging orderit is an error that is not affecting the functionality so it is not noticeable at first.

Fixed the toast width on mobile that affects the admin experience

https://github.com/user-attachments/assets/13891bf1-fb76-43f1-8665-ef4396ad8f5d

Added scroll into view the active nav link on configuration on mobile

### CODE REVIEW

File: src/includes/model/configuration/candidate-pos-model.php [hunk 1]
```diff
@@ -160,7 +160,7 @@ protected static function updateSequence($data)
                     'data_id' => $data['data_id'],
                     'sequence' => $data['sequence'],
                     'value' => $data['value'],
-                    'max_votes' => $data['max_votes'],
+                    // 'max_votes' => $data['max_votes'],
```
Comment (Carl-Tabuso) [inline src/includes/model/configuration/candidate-pos-model.php:163]: Won't this affect the selection of maximum votes per position?

Comment (Carl-Tabuso) [reply src/includes/model/configuration/candidate-pos-model.php:163]: If no, we can merge these chagnes.

Comment (make-sundae) [reply src/includes/model/configuration/candidate-pos-model.php:163]: No it is the function only called when rearranging the order of cand positions

**Comment by make-sundae** [comment]
There is still button hover style inconsistency on faq pero saka an haha

**Review by Carl-Tabuso — APPROVED** [review]

**Pull request merged**